doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -537,76 +500,85 @@ Linear Algebra Equivalents
+       ``a``, ``b``
+       where :math:`\lambda\mathbf{b}\bar{v}=\mathbf{a}\bar{v}`
 
-   * - ``[V,D]=eigs(a,k)``
-     -
-     - find the ``k`` largest eigenvalues and eigenvectors of ``a``
+   * - ``[V,D]=eigs(a,3)``
+     - ``D,V = eigs(a, k = 3)``
+     - find the ``k=3`` largest eigenvalues and eigenvectors of 2D array, ``a``
 
    * - ``[Q,R,P]=qr(a,0)``
-     - ``Q,R = scipy.linalg.qr(a)``
+     - ``Q,R = linalg.qr(a)``
      - QR decomposition
 
-   * - ``[L,U,P]=lu(a)``
-     - ``L,U = scipy.linalg.lu(a)`` or ``LU,P=scipy.linalg.lu_factor(a)``
-     - LU decomposition (note: P(MATLAB) == transpose(P(numpy)) )
+   * - ``[L,U,P]=lu(a)`` where ``a==P'*L*U``
+     - ``P,L,U = linalg.lu(a)`` where ``a == P@L@U``
+     - LU decomposition (note: P(MATLAB) == transpose(P(NumPy)))
 
    * - ``conjgrad``
-     - ``scipy.sparse.linalg.cg``
+     - ``cg``
      - Conjugate gradients solver
 
    * - ``fft(a)``
-     - ``fft(a)``
+     - ``np.fft(a)``
      - Fourier transform of ``a``
 
    * - ``ifft(a)``
-     - ``ifft(a)``
+     - ``np.ifft(a)``
      - inverse Fourier transform of ``a``
 
    * - ``sort(a)``
-     - ``sort(a)`` or ``a.sort()``
-     - sort the matrix
+     - ``np.sort(a)`` or ``a.sort(axis=0)``
+     - sort each column of a 2D array, ``a``
 
-   * - ``[b,I] = sortrows(a,i)``
-     - ``I = argsort(a[:,i]), b=a[I,:]``
-     - sort the rows of the matrix
+   * - ``sort(a, 2)``
+     - ``np.sort(a, axis = 1)`` or ``a.sort(axis = 1)``
+     - sort the each row of 2D array, ``a``
 
-   * - ``regress(y,X)``
-     - ``linalg.lstsq(X,y)``
-     - multilinear regression
+   * - ``[b,I]=sortrows(a,1)``
+     - ``I = np.argsort(a[:, 0]); b = a[I,:]``
+     - save the array ``a`` as array ``b`` with rows sorted by the first column
+
+   * - ``x = Z\y``
+     - ``x = linalg.lstsq(Z, y)``
+     - perform a linear regression of the form :math:`\mathbf{Zx}=\mathbf{y}`
 
    * - ``decimate(x, q)``
-     - ``scipy.signal.resample(x, len(x)/q)``
+     - ``signal.resample(x, np.ceil(len(x)/q))``
      - downsample with low-pass filtering
 
    * - ``unique(a)``
-     - ``unique(a)``
-     -
+     - ``np.unique(a)``
+     - returns a vector of unique values in array ``a``
 
    * - ``squeeze(a)``
      - ``a.squeeze()``
-     -
+     - remove singleton dimensions of array ``a`` Note that MATLAB will always
+       return arrays of 2D or higher while NumPy will return arrays of 0D or
+       higher
 
 .. _numpy-for-matlab-users.notes:
 
 Notes
 =====
 
 \ **Submatrix**: Assignment to a submatrix can be done with lists of
-indexes using the ``ix_`` command. E.g., for 2d array ``a``, one might
-do: ``ind=[1,3]; a[np.ix_(ind,ind)]+=100``.
+indices using the ``ix_`` command. E.g., for 2D array ``a``, one might
+do: ``ind=[1, 3]; a[np.ix_(ind, ind)] += 100``.
 
 \ **HELP**: There is no direct equivalent of MATLAB's ``which`` command,
 but the commands ``help`` and ``source`` will usually list the filename
```

Comment:
I'm pretty sure `source` does not work anymore. The best option is `inspect.getfile(func)`, I think.

Reply:
```suggestion
but the commands :func:`help`` and :func:`numpy.source` will usually list the filename
```